### PR TITLE
Add tool SCRCPY

### LINKED ISSUE #1898: [Tool] Cast Android device screen to a computer
**Platform:**
Android

**Description:**
I have recently discovered and used the tool [scrcpy](https://github.com/Genymobile/scrcpy) to mirror a device screen to a computer.

Do you are interested by a PR describing in which context this tool can be used when evaluating Android apps?

Thanks a lot in advance 😃

### PR DESCRIPTION
**Pull request by righettod**
Hi,

This PR closes #1898 

- [x] Your contribution is written in the 2nd person (e.g. you)
- [x] Your contribution is written in an active present form for as much as possible.
- [x] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [x] Your contribution has proper formatted markdown and/or code
- [x] Any references to website have been formatted as [TEXT](URL “NAME”)
- [x] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

Thanks a lot in advance 😃

### CODE REVIEW

**Comment by righettod** [comment]
Link check failed but it is not due to the PR because I have checked twice that all added links are valid 😢

File: Document/0x08-Testing-Tools.md [hunk 1]
```diff
@@ -1281,6 +1281,10 @@ Learn more about [rooting your device with Magisk](#rooting-with-magisk "Rooting
 
 A Cydia Substrate Module used to check for commonly known indications of root - <https://github.com/devadvance/rootcloakplus>
 
+### Scrcpy
+
+[Scrcpy](https://github.com/Genymobile/scrcpy "Scrcpy") provides display and control of Android devices connected on USB (or [over TCP/IP](https://www.genymotion.com/blog/open-source-project-scrcpy-now-works-wirelessly/ "Scrcpy now works wirelessly")). It does not require any root access and it works on GNU/Linux, Windows and macOS.
```
Comment (cpholguera) [inline Document/0x08-Testing-Tools.md:1286]:
```suggestion
[Scrcpy](https://github.com/Genymobile/scrcpy "Scrcpy") provides display and control of Android devices connected over USB (or [TCP/IP](https://www.genymotion.com/blog/open-source-project-scrcpy-now-works-wirelessly/ "Scrcpy now works wirelessly")). It does not require any root access and it works on GNU/Linux, Windows and macOS.
```

**Review by cpholguera — CHANGES_REQUESTED** [review]
Only one little suggestion :)

**Comment by cpholguera** [comment]
same here, I've changed

This PR cover the proposal described in #1898 

to

This PR closes #1898 

to let GitHub do its magic ✨

**Comment by righettod** [comment]
I have commited your suggestion, it is great 😃

**Review by cpholguera — APPROVED** [review]
Thanks for adding this @righettod!

**Comment by righettod** [comment]
You are welcome @cpholguera , it is always a pleasure to contribute to this amazing project 💯 and exchange with you 👍

**Comment by cpholguera** [comment]
Now that you're into the cleanup mood, which we really appreciate, maybe you'd like to take over this one: https://github.com/OWASP/owasp-mstg/issues/1839

What do you think? Maybe #1838 as well? If yes, please let me know in order to assign them to you.

**Comment by righettod** [comment]
Yes, can assign it to me, I will use the following searches as identification points:
- [Needle](https://github.com/OWASP/owasp-mstg/search?q=needle)
- [Drozer](https://github.com/OWASP/owasp-mstg/search?q=drozer)

I will try to work on it coming tuesday, otherwise next week end 😃

**Comment by cpholguera** [comment]
Thank you very much @righettod!

**Pull request merged**